testnet/stacks-node/src/nakamoto_node/peer.rs

```diff
@@ -202,6 +202,21 @@ impl PeerThread {
         }
     }
 
+    fn check_stackerdb_reload(&mut self) {
```

Comment:
Minor nit: This is doing a check and then maybe doing some work based on that check.  Maybe this should be called `stackerdb_refresh()` or something?